calculator.py

```diff
@@ -40,45 +40,118 @@
 
 
 """
+# import pysnooper
 
-
+# @pysnooper.snoop()
 def add(*args):
     """ Returns a STRING with the sum of the arguments """
 
-    # TODO: Fill sum with the correct value, based on the
+    # TODONE: Fill sum with the correct value, based on the
     # args provided.
-    sum = "0"
 
-    return sum
+    my_sum = sum(map(int,args))
+
+    return str(my_sum)
+
+# TODONE: Add functions for handling more arithmetic operations.
+
+def subtract(*args):
+    """ Returns a STRING with the difference of the arguments """
+
+
+    a = int(args[0])
+    b = int(args[1])
+    difference = a - b
+
+    return str(difference)
+
+def multiply(*args):
+    """ Returns a STRING with the product of the arguments """
+
+    product = 1
 
-# TODO: Add functions for handling more arithmetic operations.
+    for arg in args:
+        product *= int(arg)
+
+    return str(product)
+
+
+def divide(*args):
+    """ Returns a STRING with the quotient of the arguments """
+
+
+    dividend = int(args[0])
```

Comment:
Same note about handling multiple arguments here, don't assume that it will just be two 😄